dashboard/src/components/BootsTable/BootsTable.tsx

```diff
@@ -150,7 +152,47 @@ const TableCellComponent = ({
   );
 };
 
+const TableRowComponent = ({
+  row,
+  index,
+  currentLog,
+  getRowLink,
+  openLogSheet,
+}: {
+  row: Row<TestByCommitHash>;
+  index: number;
+  currentLog?: number;
+  getRowLink: (testId: TestHistory['id']) => LinkProps;
+  openLogSheet: (index: number) => void;
+}): JSX.Element => {
+  const className = useMemo(
+    () => (index === currentLog ? 'bg-lightBlue' : undefined),
+    [index, currentLog],
+  );
```

Comment:
we don't need `useMemo` for this

memo are for expensive calculations and unstable references

an equality comparisson is a preach cheap operation and the result is a primitive which javascript compares by equality